annoto/annoto.py

```diff
@@ -176,6 +175,7 @@ def _base_view(self, context=None):
         course_display_name = course.display_name
         user = self._get_user()
         if user and self.discussions_scope == 'cohort':
+            from openedx.core.djangoapps.course_groups.cohorts import get_cohort
```

Comment:
Could we place this at the top of the function? I know it's conditionally needed but it makes it easier to find.

Reply:
Done. I also squashed the commits so it's ready for merge. :) CC @annoto-campus